Tools/fix_code_style.sh

```diff
@@ -1,4 +1,13 @@
-#!/bin/sh
+#!/bin/bash
+
+ASTYLE_VER=`astyle --version`
+ASTYLE_VER_REQUIRED="Artistic Style Version 2.05.1"
```

Comment:
https://github.com/dagar/astyle/releases/tag/2.05.1

I can transfer ownership of this repo to PX4 and update the error message to point users at the right version.